app/controllers/admin/bulk_republishing_controller.rb

```diff
@@ -3,6 +3,43 @@ class Admin::BulkRepublishingController < Admin::BaseController
 
   before_action :enforce_permissions!
 
+  def new_by_type
+    @republishable_content_types_select_options = republishable_content_types_select_options
+  end
+
+  def new_by_type_redirect
+    redirect_to(admin_bulk_republishing_by_type_confirm_path(params[:content_type]))
+  end
+
+  def confirm_by_type
+    content_type = republishable_content_types.find { |type| type == params[:content_type].underscore.camelcase }
+    return render "admin/errors/not_found", status: :not_found unless content_type
+
+    @bulk_content_type_metadata = bulk_content_type_metadata.fetch(:all_by_type)
+    @content_type = content_type
+    @republishing_event = RepublishingEvent.new
+  end
+
+  def republish_by_type
+    content_type = republishable_content_types.find { |type| type == params[:content_type].underscore.camelcase }
+    return render "admin/errors/not_found", status: :not_found unless content_type
+
+    bulk_content_type_key = :all_by_type
+    @bulk_content_type_metadata = bulk_content_type_metadata.fetch(bulk_content_type_key)
+    action = "#{@bulk_content_type_metadata[:name].upcase_first} '#{params[:content_type]}' have been queued for republishing"
```

Comment:
Would it be better to use `content_type` rather than `params[:content_type]` in the action here to get the class-y casing (how we present it to users in the select field and docs)?